apps/zchat-api/src/features/zrocket/zrocket.controller.ts

```
@@ -0,0 +1,149 @@
+                    {
+                        name: 'systemMessages',
+                        filter: { t: 'system' },
+                        description: 'System-generated messages for events'
+                    }
+                ],
+                fromParticipantsCollection: [
+                    {
+                        name: 'userParticipants',
+                        filter: { type: 'user' },
+                        description: 'Human user participants'
+                    },
+                    {
+                        name: 'botParticipants',
+                        filter: { type: 'bot' },
+                        description: 'Bot participants with configuration'
+                    }
+                ]
+            },
+            features: [
+                'Real-time filtering: Documents are routed to appropriate Zero tables based on filter criteria',
+                'Projection support: Only relevant fields are synced to each table',
+                'Update routing: When documents change, they may be inserted/removed from tables based on new filter matches',
+                'Efficient change streams: Single MongoDB change stream watches all source collections'
+            ],
+            endpoints: [
+                'GET /zrocket/demo-info - This information',
+                'POST /zrocket/seed-data - Seeds sample data for testing',
+                'GET /zrocket/collections/{collection} - View raw MongoDB collection data',
+                'GET /zrocket/tables - List all discriminated Zero tables'
+            ]
+        };
+    }
+
+    @Post('seed-data')
+    @ApiOperation({
+        summary: 'Seed sample data for the discriminated union demo',
+        description: 'Creates sample rooms, messages, participants, and users in MongoDB to demonstrate the discriminated union functionality'
+    })
+    @ApiResponse({ status: 201, description: 'Sample data seeded successfully' })
+    async seedSampleData() {
```

Allow optionally passing custom seed data in the request body. Fall back to hardcoded data only if not provided